content/doc/book/managing/nodes.adoc

```diff
@@ -14,23 +14,23 @@ endif::[]
 
 = Managing Nodes
 
-== Launch Java Web Start slave agent via Windows Scheduler
+== Launch Java Web Start agent via Windows Scheduler
```

Comment:
Using "inbound" instead of "Java Web Start" because Java Web Start is not available for Java 11.
```suggestion
== Launch inbound agent via Windows Scheduler
```